lib/pure/complex.nim

```diff
@@ -11,7 +11,38 @@
 ## and basic mathematical operations on them.
 ##
 ## Complex numbers are currently generic over 64-bit or 32-bit floats.
-
+##
+## Here is an example that shows how to use this module:
+##
+## .. code-block:: nim
```

Comment:
use runnableExamples

Reply:
I saw that there was a clause `runnableExample` in the [source code of the module](https://github.com/nim-lang/Nim/blob/88a26721313a78c469f7ca09ecff70c28e87c90f/lib/pure/complex.nim#L46-L65), but for some reason it does not show up here: https://nim-lang.org/docs/complex.html, which is the place where I usually look up for documentation. I felt it would have been useful to have a real example appear there.

Is there a problem with the way that page has been rendered by GitHub Pages, or are you suggesting that I just copy-and-paste that example in my `code-block` so that it appears there? The documentation I checked (the [same page you linked](https://nim-lang.github.io/Nim/contributing.html#documentation)) does not state clearly when runnable examples are included in HTML docs and when they are not; empirically,l it seems to me that they do only if the are within a procedure/function, is this the expected behavior?

I decided to use a `code-block` just because I discovered that this is used in [other parts of the standard library](https://github.com/nim-lang/Nim/blob/version-1-4/lib/pure/selectors.nim#L217-L225).

Reply:
Those docs you linked are outdated (they're not the devel docs), the latest docs can be found here: https://nim-lang.github.io/Nim/complex.html. Regarding `code-block`, eventually all occurences of it should be replaced by `runnableExamples`.

Reply:
Ok, I see. I just followed the links starting from the home site, as shown by this animation:

![nim-docs](https://user-images.githubusercontent.com/377795/122039630-28235900-cdd7-11eb-910b-ac78395a14d6.gif)

At this point, I think it is not worth to keep this PR open. But when will these devel docs become accessible from the front page?

Reply:
I assume they'll become available with the next major release (1.6).

Reply:
I see. It's a pity, just yesterday I advertised Nim to my students (I am currently teaching a course in numerical analysis and software engineering) and was willing to polish a bit the documentation of the libraries my students are likely to try. Guess I should first tell them to use the documentation link you passed me instead of the official one.